runtime/sema/type.go

```diff
@@ -5581,7 +5659,7 @@ func checkSubTypeWithoutEquality(subType Type, superType Type) bool {
 					// TODO: once interfaces can conform to interfaces, include
```

Comment:
```suggestion
```

There are some other "once interfaces can conform to interfaces" comments in the code-base, maybe double-check they are addressed and remove